### ISSUE TITLE
Angular + Gridder = ng-click directive doesn't work

### ISSUE THREAD

**Issue by bharathitman** [oriongunning/gridder#48]
I know this is a jquery only plugin. I am using gridder with angular. The 'expand template' (The expanding preview) of gridder includes  buttons. However, these buttons do not propogate ng-click (similar to onclick, but calls a function in the controller instead - think mvc, view calling the controller) events at all. Seems to be a weird issue.

**Comment by oriongunning** [comment]
Yes, from what I understand, all content need to be compiled. They must some way to compile dynamically generated content? I'm not familiar with angular, sorry.

**Comment by sateesh2499** [comment]
Hi, I am also trying to use gridder with angularjs. Is there any workaround to make it work?
